src/Icon/Container.jsx

```diff
@@ -9,6 +9,10 @@ export const Container = () => {
         <span>aa</span>
```

Comment:
nit: do we need the curly brackets and explicit return?

Reply:
no that's just used for the text. You can see how it works [here](https://marshmallow-insurance.github.io/smores-react/?path=/story/icon--collection) but it's essentialy a row of `iconRenderText      icon`